src/ui/Facet/Facet.ts

```diff
@@ -674,9 +674,8 @@ export class Facet extends Component {
      */
     enableResponsiveMode: ComponentOptions.buildBooleanOption({ defaultValue: true, section: 'ResponsiveOptions' }),
     responsiveBreakpoint: ComponentOptions.buildNumberOption({
-      defaultValue: 800,
       deprecated:
-        'This option is exposed for legacy reasons. It is not recommended to use this option. Instead, use `ResponsiveComponents` methods exposed on the `SearchInterface`.'
+        'This option is exposed for legacy reasons. It is not recommended to use this option. Instead, use `SearchInterface.options.responsiveMediumBreakpoint` options exposed on the `SearchInterface`.'
```

Comment:
(...) Instead, use the [`responsiveSmallBreakpoint`]{@link SearchInterface.options.responsiveSmallBreakpoint} and/or [`responsiveMediumBreakpoint`]{@link SearchInterface.options.responsiveMediumBreakpoint} options of the [`SearchInterface`]{@link SearchInterface} component.